content/learning-paths/smartphones-and-mobile/android_webgpu_dawn/2-env-setup.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Setting up development environment
+weight: 3
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+## Set up your development environment
+
+In this Learning Path, you will learn how to
+
+* Integrate Dawn (WebGPU) in the application.
+* Use the APIs to render a simple 3D object.
+* Profile and analyze the application.
+
+Arm has worked with Dawn team to optimize the uploading data to GPU buffers for Mali GPUs. Arm has implemented a **fast path** mechanism wherein Vertex Queue starts processing in parallel while an earlier Fragment Queue is being processed.
```

Comment:
I kept it here initially, since the pre-requisite mentions target audience has some Android application development experience. Moving to "Profiling" section makes more sense

Reply:
Move to Dawn section above (line 39 in webgpufundamentals) maybe?